www/js/VisitorChat/5.0/ChatBase.js.php

```diff
@@ -574,7 +619,7 @@
     updateChatContainerWithHTML:function (selector, html, sendAlerts) {
         //$.parseHTML(html)[0].outerHTML is used to compare the rendered html (the browser can change quotes, etc)
         //It just makes the comparison more accurate (at the cost of a little speed)
-        if ($(selector).html() === $.parseHTML(html)[0].outerHTML) {
+        if (selector.innerHTML === $.parseHTML((html)[0].outerHTML)) {
```

Comment:
"selector" by itself probably doesn't work.  Do you mean "document.querySelector(selector)" ?